gateway-js/src/config.ts

```diff
@@ -20,9 +24,7 @@ export type Experimental_DidResolveQueryPlanCallback = ({
   readonly queryPlan: QueryPlan;
   readonly serviceMap: ServiceMap;
   readonly operationContext: OperationContext;
-  readonly requestContext: GraphQLRequestContextExecutionDidStart<
-    Record<string, any>
-  >;
+  readonly requestContext: GraphQLRequestContext<Record<string, any>>;
```

Comment:
This is a change in type because we don't need the guarantee of the other fields here, only the request. Plus this now matches up with the other usages